en/admin/db_manage.rst

```diff
@@ -57,9 +57,13 @@ The volumes of CUBRID database are classified as permanent volume, temporary vol
 *   In the permanent volumes,
 
     *   there are data volumes, that usually store permanent data, but can also store temporary data.
-    *   there are log volumes, that can be further classified as: one active log, archive logs and one background archiving log.
+    *   there is the active log volume, that includes late changes of log records.
     
-*   In the temporary volumes, only temporary data is stored.
+*   In the temporary volumes,
+
+    *   there are temporary temp volumes, that store temporary data.
+    *   there are archive log volumes, that store exhausted logs from the active log.
+    *   there is one background archive log volume, that use to store archive log in the background.
```

Comment:
```suggestion
    *   there is one background archive log volume, that is used to store archive log in the background.
```

or

```suggestion
    *   there is one background archive log volume, that store archive log in the background.
```

Reply:
`here is one background archive log volume, that store's' archive log in the background.` I will append 's' behind the store. Would it be better?